test/NonSilo.Tests/ConfigTests.cs

```diff
@@ -347,58 +186,6 @@ public void ServerConfig_TraceFilePattern_Blank()
         }
 
         [Fact, TestCategory("Functional"), TestCategory("Config"), TestCategory("Logger")]
-        public void ClientConfig_LogConsumers()
-        {
-            LogManager.UnInitialize();
-
-            string filename = "Config_LogConsumers-ClientConfiguration.xml";
-
-            var cfg = ClientConfiguration.LoadFromFile(filename);
-            Assert.Equal(filename, cfg.SourceFile);
-
-            LogManager.Initialize(cfg);
-            Assert.Collection(LogManager.LogConsumers,
-                lc => Assert.IsAssignableFrom<TelemetryLogConsumer>(lc),
-                lc => Assert.IsAssignableFrom<DummyLogConsumer>(lc));
-        }
-
-        [Fact, TestCategory("Functional"), TestCategory("Config"), TestCategory("Logger")]
-        public void ServerConfig_LogConsumers()
-        {
-            LogManager.UnInitialize();
-
-            string filename = "Config_LogConsumers-OrleansConfiguration.xml";
-
-            var cfg = new ClusterConfiguration();
-            cfg.LoadFromFile(filename);
-            Assert.Equal(filename, cfg.SourceFile);
-
-            LogManager.Initialize(cfg.CreateNodeConfigurationForSilo("Primary"));
-
-            Assert.Collection(LogManager.LogConsumers,
-                lc => Assert.IsAssignableFrom<TelemetryLogConsumer>(lc),
-                lc => Assert.IsAssignableFrom<DummyLogConsumer>(lc));
-        }
-
-        [Fact, TestCategory("Functional"), TestCategory("Config"), TestCategory("Logger")]
-        public void ClientConfig_TelemetryConsumers()
```

Comment:
most of the tests removed here (except for very few) are still relevant for the legacy package or for the telemetry consumer abstraction that we are still preserving. Please update how they are set up, but please bring them back, otherwise we loose all coverage